newsletters/20190925-zero-trust.md

```diff
@@ -0,0 +1,46 @@
+Hey there!
+
+_TL;DR: Learnk8s Academy discount ends today, and we published an article on securing Secrets in Kubernetes._
+
+**1. Last call**
+
+The Learnk8s Academy is live and to celebrate the release we discounted the most popular bundle for a week!
+
+**Today is the last day.**
+
+There are still a handful of hours before we close the promotion.
```

Comment:
Maybe defining an absolute time if there is a strict deadline? Like, today 23:59 UTC